app/src/users/templates/users/profile.html

```diff
@@ -169,7 +169,9 @@ <h5>
             </h5>
           {% else %}
             <div class="card-content">
-              {% if not registration_open %}
+              {% if zosia.registration_suspended %}
+              <p>{% trans "Registration is suspended"%}</p>
```

Comment:
I'd add more info, like: "Registration is suspended, please wait for further information".